docs/features.md

```diff
@@ -35,3 +35,55 @@ PSDocs runs on MacOS, Linux and Windows.
 PowerShell makes it easy to integrate PSDocs into popular CI systems.
 To install, use the `Install-Module` cmdlet within PowerShell.
 For installation options see [install instructions](install-instructions.md).
+
+## Frequently Asked Questions (FAQ)
+
+### Can PSDocs read from metadata.json?
+
+The Azure Quickstart Templates repository uses an additional `metadata.json` to store template metadata.
+PSDocs doesn't require a `metadata.json` file to exist but will fallback to this file if it exists.
```

Comment:
Should we just mention that the `metadata.md` is case insensitive/sensitive?

Reply:
Also for other properties within metadata.json?

Reply:
File name would be on Linux. Properties of `metadata.json` aren't case sensitive.